tests/models/User.php

```diff
@@ -1,14 +1,15 @@
 <?php
 
 use Jenssegers\Mongodb\Eloquent\Model as Eloquent;
+use Jenssegers\Mongodb\Eloquent\HybridRelations;
 use Illuminate\Auth\Authenticatable;
 use Illuminate\Auth\Passwords\CanResetPassword;
 use Illuminate\Contracts\Auth\Authenticatable as AuthenticatableContract;
 use Illuminate\Contracts\Auth\CanResetPassword as CanResetPasswordContract;
 
 class User extends Eloquent implements AuthenticatableContract, CanResetPasswordContract
 {
-    use Authenticatable, CanResetPassword;
```

Comment:
This should not be needed right? Since `Eloquent` had this trait as well?

Reply:
Not sure what you mean, but I added the `HybridRelations` trait for the User model as using that trait immediately gives an error without this fix. In the first commit I used 2 `use` statements which I replaced to be 1 statement, therefor this line shows as deleted.

I dont think the Eloquent model class (either mongodb's or laravel's) implements one of these traits already

Reply:
`Jenssegers\Mongodb\Eloquent\Model` already has the `HybridRelations` trait, which the model extends from.

Reply:
Ah ok. Thanks for merging.